metricbeat/module/http/json/_meta/data.json

```diff
@@ -1,4 +1,4 @@
-{
+[{
```

Comment:
This part is not going to change. In case of an array, each document will be sent as a single vent in the previous format.